src/components/footer/example-footer-with-additional-logo.njk

```diff
@@ -0,0 +1,89 @@
-3.09 1.16-3.09 2.78 0 .85.42 1.65 1.13 2.13-1.08.74-1.62 1.52-1.62 2.38 0 .77.52 1.45 1.27 1.65-.75.23-1.28.91-1.3 1.69 0 1.3 1.59 2.35 3.55 2.35 2.32 0 4.06-1.52 4.06-3.61 0-1.55-.65-2.13-2.32-2.13l-3.17.05c-.93 0-1.11-.22-1.11-.61s.42-.95 1.21-1.45c.45.22.95.31 1.45.29 1.77 0 3.12-1.12 3.12-2.65 0-.63-.21-1.24-.58-1.74.1-.24.15-.27.25-.27Zm-2.85 4.06c-.87 0-1.11-.37-1.11-1.69 0-2.48.17-2.67 1.16-2.67.87 0 1.11.17 1.11 1.86 0 1.39-.05 1.79-.19 2.06-.22.31-.59.48-.97.45Zm-.07 7.43c-1.35 0-2.26-.76-2.26-1.91 0-.52.21-1.02.58-1.39h.08c.36.05.72.06 1.08.05h.99c.23.01.46.01.68 0h1.13c.83 0 .98.24.98.71-.02 1.4-1.45 2.55-3.27 2.55Z" style="fill:#001f5b"/><path d="M70.45 46.48v-.1C70.3 44.32 68.98 43 67.04 43s-3.65 1.89-3.65 4.02 1.77 4.01 3.71 4.01c1.63.02 3.01-1.17 3.25-2.78v-.1h-.65v.07c-.35 1.39-1.3 2.19-2.45 2.19a1.59 1.59 0 0 1-1.45-.78c-.22-.4-.25-.77-.25-2.61v-.46h4.9v-.06Zm-3.46-2.87c.5-.07.98.22 1.16.69.17.52.22 1.07.15 1.61h-2.78v-.34c0-1.42.4-1.96 1.48-1.96Zm11.22 6.41c-.35.29-.37.29-.5.29-.32 0-.35 0-.35-1.67v-2.55c0-1.64 0-1.79-.46-2.26a3.2 3.2 0 0 0-2.32-.83c-1.65 0-3.07.93-3.07 2.01-.02.48.36.88.84.9h.03c.46.01.85-.35.87-.82s0-.06 0-.08c0-.25-.11-.48-.29-.65q-.07-.1-.07-.15c0-.29.69-.71 1.35-.71.35-.03.7.1.96.34.19.22.24.37.24 1.48v1.23c-2.9.07-4.5.98-4.5 2.61 0 1.13.84 1.89 2.11 1.89.97 0 1.88-.49 2.43-1.3.11.79.82 1.36 1.62 1.3.52 0 1.02-.21 1.39-.58l.05-.05-.35-.41Zm-2.72-2.97v.29c.05.63-.03 1.26-.25 1.86-.24.5-.74.83-1.3.85-.56.04-1.04-.37-1.09-.93v-.21c0-1.21.82-1.77 2.64-1.86ZM86.24 38h-3.16v.54h1.3v5.85a2.438 2.438 0 0 0-2.18-1.3c-1.74 0-3.25 1.81-3.25 3.97s1.5 4.06 3.31 4.06c.89-.01 1.7-.51 2.11-1.3v1.16h3.14v-.56h-1.27V38Zm-3.9 12.43c-.43-.01-.81-.25-1.01-.64-.19-.39-.25-.78-.25-2.5 0-1.95.08-2.68.27-3.07.2-.34.56-.55.95-.58 1.13 0 1.95 1.42 1.95 3.41s-.8 3.37-1.92 3.37Zm12.7-.41c-.35.29-.37.29-.49.29-.32 0-.35 0-.35-1.67v-2.55c0-1.64 0-1.79-.46-2.26a3.2 3.2 0 0 0-2.32-.83c-1.64 0-3.07.93-3.07 2.01-.02.48.36.88.84.9h.06c.47.03.87-.33.9-.8v-.1c0-.25-.11-.48-.29-.65q-.08-.1-.08-.15c0-.29.69-.71 1.35-.71.35-.02.7.1.95.34.21.22.25.37.25 1.48v1.23c-2.9.07-4.49.98-4.49 2.61 0 1.13.83 1.89 2.11 1.89.97 0 1.88-.49 2.43-1.3.11.79.82 1.36 1.62 1.3.52 0 1.03-.21 1.39-.58l.05-.05-.35-.41h-.06Zm-2.73-2.97v.29c.05.63-.03 1.26-.24 1.86-.24.51-.74.84-1.3.85-.56.04-1.04-.37-1.09-.93v-.22c0-1.2.82-1.75 2.63-1.85Zm5.5-6.45c.63-.02 1.14-.53 1.16-1.16 0-.64-.5-1.17-1.15-1.17h-.02c-.61-.03-1.13.43-1.16 1.04s0 .08 0 .12c0 .63.51 1.15 1.14 1.16h.03Zm.98 2.6h-3.14v.56h1.25v6.53h-1.25v.56h4.39v-.56h-1.25V43.2zm4.39 2.65c-1.52-.27-1.79-.5-1.79-1.01.1-.77.8-1.32 1.57-1.23 1.05 0 1.93.78 2.06 1.82v.08h.46l-.07-2.4h-.42l-.27.58c-.5-.42-1.13-.65-1.79-.65-1.28-.04-2.35.97-2.39 2.25v.16c0 1.52.69 2.16 2.65 2.5 1.45.22 1.72.58 1.72 1.13a1.49 1.49 0 0 1-1.59 1.39h-.06c-1.2 0-1.89-.73-2.35-2.53v-.07h-.42l.05 3.01h.35l.42-.56c.54.5 1.25.77 1.98.76 1.4 0 2.53-1.14 2.53-2.53v-.19c0-1.45-.73-2.18-2.63-2.5ZM33.65 25.99h71.94v.27H33.65z" style="fill:#001f5b"/></svg>',
```

Comment:
I think for the example we would want to use a black and white image to match the ons one